hydromt/gis_utils.py

```diff
@@ -553,3 +558,46 @@ def to_geographic_bbox(bbox, source_crs):
         bbox = Transformer.from_crs(source_crs, target_crs).transform_bounds(*bbox)
 
     return bbox
+
+
+def prepare_pyogrio_reader_filters(
```

Comment:
I think this method could be more widely used than for pyogrio input. Perhaps renaming it to `parse_bbox` or something more generic would make that more clear.